Jenkinsfile

```diff
@@ -76,3 +95,15 @@ void withMavenEnv(List envVars = [], def body) {
         body.call()
     }
 }
+
+void renameFiles(def files, String prefix) {
+    for(i = 0; i < files.length; i++) {
```

Comment:
Does `File.renamteTo` from the GDK not work properly in Pipeline or require a Script Security Approval?

This is fine, but seems kind of hacky; without a comment it's unclear to me whether this was done as a workaround or just because?